reo/validators.py

```diff
@@ -814,6 +815,38 @@ def remove_nones(self, object_name_path, template_values=None, real_values=None,
                         if input_isDict == False:
                             self.input_as_none.append([name, object_name_path[-1] + ' (number {})'.format(number)])
 
+    
+    def check_constraint_ranges(self, object_name_path, template_values=None, real_values=None, number=1, input_isDict=None):
```

Comment:
Should `input_isDict` be a `bool`?

Reply:
It will effectively always be a bool since this is the only place the function is ever called here: 
https://github.com/NREL/REopt_Lite_API/blob/04310010bfd6829cc37994a6a00b84b64bd55349/reo/validators.py#L656

Reply:
Ok then should the default value also be a `bool`?

Reply:
We could make default for input_isDict a bool but I don't see a particular reason for it. With default being None it may help to debug cases where input_isDict isn't set for some reason (we know True/False was explicitly set somewhere)